### DISCUSSION TITLE
How to connect Make with budibase

### DISCUSSION THREAD

**Discussion by AryanGaur14** [Budibase/budibase#13326] [Help] [answered]
I want to learn how to connect Make with Budibase because Budibase is not available on the Make platform so, how can we do that.

**Comment by andz-bb** [comment, marked as answer]
hi again @AryanGaur14, you'll need to use webhooks again. [See this guide in our documentation for reference.](https://docs.budibase.com/docs/make-integromat) If you have any trouble with this feel free to ask here!

**Reply by AryanGaur14** [reply]
Thank you so much for providing the answer! Your insight really helped.

**Reply by AryanGaur14** [reply]
i have one more question regarding this could please explain 
{
   "value": "{{ trigger.fields.test  }}"
}
in payload like i understood what is "test" here but not same with trigger and fields
